resources/views/peilingen/peiling.blade.php

```diff
@@ -0,0 +1,14 @@
+<Peiling
+	class="vue-context"
+	:id="{{ $peiling->id }}"
+	:titel="'{{ $peiling->titel }}'"
+	:beschrijving="'{!! htmlspecialchars(\CsrDelft\view\bbcode\CsrBB::parse($peiling->beschrijving)) !!}'"
```

Comment:
Nu gaat het kapot als er een line break in de (gerenderde) beschrijving voorkomt (bijvoorbeeld in peiling 110). Misschien werkt het om {!! json_encode(htmlspecialchars(\CsrDelft\view\bbcode\CsrBB::parse($peiling->beschrijving))) !!} te doen in plaats van '{!! htmlspecialchars(\CsrDelft\view\bbcode\CsrBB::parse($peiling->beschrijving)) !!}'

Reply:
`json_encode` hacks 😍

Reply:
Werkt trouwens, thanks!

Reply:
Ok, nice. Doe het dan ook maar bij alle andere velden (in ieder geval bij alle string velden), want hoewel daar meestal geen multiline strings voorkomen kunnen ze wel worden ingevoerd op die manier en daar kan potentieel een XSS attack mee worden uitgevoerd.